code/parachain/frame/staking-rewards/src/test/mod.rs

```diff
@@ -796,6 +797,7 @@ fn unstake_in_case_of_not_zero_claims_and_early_unlock_should_work() {
 		process_and_progress_blocks::<StakingRewards, Test>(1);
 
 		mint_assets([BOB], [PICA::ID], PICA::units(200));
+		mint_assets([BOB], [XPICA::ID], XPICA::units(200));
```

Comment:
you should not be minting any xtokens in the tests, xtokens are only to be minted by the pallet